### PR TITLE
Preview

### PR DESCRIPTION
**Pull request by raphaeldieu**
Set a preview page

### CODE REVIEW

File: app/assets/javascripts/angular/controller/forms/previewForm.js.erb [hunk 1]
```diff
@@ -7,20 +7,36 @@ app.controller('PreviewFormCtrl', ['$scope', '$http', '$auth', '$location', '$ro
...
     $scope.contents = response.contents;
+
+    //Get all the options for every contents
     $scope.contents.forEach(function(content) {
       if (content.options !== null) {
-        console.log('content.options :',content.options)
-        $scope.options = content.options
+
+        console.log('content.options :',content.options);
+        $scope.options = content.options;
```
Comment (h8rry) [inline app/assets/javascripts/angular/controller/forms/previewForm.js.erb]: this doesn't have any use

File: app/assets/javascripts/angular/controller/forms/updateForms.js.erb [hunk 2]
```diff
@@ -78,14 +78,13 @@ app.controller('FormsUpdateCtrl', ['$scope', '$http', '$auth', '$location', '$ro
   }
   
   $scope.options = []; //List of choices in a dropdown menu or checkboxes
```
Comment (ggharry) [inline app/assets/javascripts/angular/controller/forms/updateForms.js.erb]: This?

File: app/assets/javascripts/angular/controller/forms/updateForms.js.erb [hunk 3]
```diff
@@ -135,7 +134,7 @@ app.controller('FormsUpdateCtrl', ['$scope', '$http', '$auth', '$location', '$ro
     console.log($scope.updatedContent);
     $http.put(url + '/contents/' + field.id,$scope.updatedContent).success(function(response) {
       $scope.editPanelVisible = false;
-      $scope.options = []
+      $scope.options = [];
```
Comment (ggharry) [inline app/assets/javascripts/angular/controller/forms/updateForms.js.erb]: This?

**Comment by h8rry** [comment]
https://github.com/WDI-HK-8/Formidable-web/blob/preview/app/assets/javascripts/angular/controller/forms/updateForms.js.erb#L93

should be

```
$scope.options = $scope.field.options
```

**Comment by h8rry** [comment]
More like this: https://github.com/WDI-HK-8/Formidable-web/blob/preview/app/assets/javascripts/angular/controller/forms/updateForms.js.erb#L31

Please fix all problems like this throughout all apps...

File: app/assets/javascripts/angular/controller/forms/previewForm.js.erb [hunk 4]
```diff
@@ -7,20 +7,36 @@ app.controller('PreviewFormCtrl', ['$scope', '$http', '$auth', '$location', '$ro
...
+
+        content.options.forEach(function(option) {
```
Comment (h8rry) [inline app/assets/javascripts/angular/controller/forms/previewForm.js.erb:22]: @ctlong @raphaeldieu @yohanology 

If you need to do a lot of these things, use [underscore.js](http://underscorejs.org/)

**Comment by raphaeldieu** [comment]
updated

File: app/assets/templates/forms/modalCreateForm.html [hunk 5]
```diff
@@ -0,0 +1,24 @@
+<div class="modal fade">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" ng-click="close(false)" data-dismiss="modal" aria-hidden="true">&times;</button>
```
Comment (h8rry) [inline app/assets/templates/forms/modalCreateForm.html:5]: Where is this `close` function defined?

Comment (h8rry) [inline app/assets/templates/forms/modalCreateForm.html:5]: Why do you need to pass in a param?

**Comment by h8rry** [comment]
@raphaeldieu please review

**Pull request merged**